src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -181,12 +182,16 @@ class ActivityFeed extends React.Component<Props, State> {
         }
     };
 
-    handleToggleHeaderShadow = () => {
-        if (this.feedContainer) {
-            const shouldHaveShadow = this.feedContainer.scrollTop !== 0;
+    handleFeedScroll = (event: SyntheticEvent<HTMLDivElement>): void => {
+        const { target } = event
```

Comment:
This value can be destructured inline and is missing a semicolon. We lint on push, so please avoid using `--no-verify` when making/pushing commits.